test/gtest/aca_test_zeta_programming.cpp

```diff
@@ -0,0 +1,263 @@
+// Copyright 2019 The Alcor Authors.

```

Comment:
per last community meeting, please take a look at Rio's change based on WanChen edit #179 which is merged today. Looks like it is doing similar operations as your change here. One option is to remove your change in this file and the team can start the test modification from master.

@HuaqingTu , @PikaPikaW